example/lib/screens/settings.dart

```diff
@@ -18,20 +21,19 @@ class Settings extends StatelessWidget {
         body: ListView(
             children: ListTile.divideTiles(
               context: context,
-              tiles: [
-                StreamBuilder(
-                    stream: _airshipBloc.notificationsEnabledStream,
-                    builder: (context, snapshot) { return SwitchListTile(
+              tiles: [ FutureBuilder(
+                  future: Airship.userNotificationsEnabled,
+                  builder: (context, snapshot) {
+                    return SwitchListTile(
                       title: Text('Push Enabled',
                         style: Styles.settingsPrimaryText,),
-                      value: snapshot.hasData ? snapshot.data : false,
-                      onChanged: (bool value){
-                        _airshipBloc.notificationsEnabledSetSink.add(value);
-                        },
-                    );
-                    }),
-                StreamBuilder(
-                    stream: _airshipBloc.namedUserStream,
+                      value: snapshot.data ?? false,
```

Comment:
Should this have changed to `snapshot.data`? I think that can be a different type than boolean, but I'm not sure what effect that would have.

Reply:
this future can only result in a null or a boolean, if it's null is coalesces to false